test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -55,6 +57,48 @@ public void TestPfiRegressionOnDenseFeatures()
             Done();
         }
 
+        /// <summary>
+        /// Test PFI Regression for Dense Features with a model loaded from disk
+        /// </summary>
+        [Fact]
```

Comment:
Similar to the functional tests - this could be a `[Theory]` instead, to not have to duplicate so much logic.

Reply:
Fixed in my last commit.